src/store-api/src/sst_entry.rs

```diff
@@ -56,6 +69,102 @@ pub struct ManifestSstEntry {
     pub sequence: Option<u64>,
 }
 
+impl ManifestSstEntry {
+    /// Returns the schema of the manifest sst entry.
+    pub fn schema() -> SchemaRef {
+        use datatypes::prelude::ConcreteDataType as Ty;
+        Arc::new(Schema::new(vec![
+            ColumnSchema::new("table_dir", Ty::string_datatype(), false),
+            ColumnSchema::new("region_id", Ty::uint64_datatype(), false),
+            ColumnSchema::new("table_id", Ty::uint32_datatype(), false),
+            ColumnSchema::new("region_number", Ty::uint32_datatype(), false),
+            ColumnSchema::new("region_group", Ty::uint8_datatype(), false),
+            ColumnSchema::new("region_sequence", Ty::uint32_datatype(), false),
+            ColumnSchema::new("file_id", Ty::string_datatype(), false),
+            ColumnSchema::new("level", Ty::uint8_datatype(), false),
+            ColumnSchema::new("file_path", Ty::string_datatype(), false),
+            ColumnSchema::new("file_size", Ty::uint64_datatype(), false),
+            ColumnSchema::new("index_file_path", Ty::string_datatype(), true),
+            ColumnSchema::new("index_file_size", Ty::uint64_datatype(), true),
+            ColumnSchema::new("num_rows", Ty::uint64_datatype(), false),
+            ColumnSchema::new("num_row_groups", Ty::uint64_datatype(), false),
+            ColumnSchema::new("min_ts", Ty::timestamp_nanosecond_datatype(), false),
+            ColumnSchema::new("max_ts", Ty::timestamp_nanosecond_datatype(), false),
```

Comment:
Please check the nullable flag.

```suggestion
            ColumnSchema::new("min_ts", Ty::timestamp_nanosecond_datatype(), true),
            ColumnSchema::new("max_ts", Ty::timestamp_nanosecond_datatype(), true),
```